src/Features/CSharp/Portable/IntroduceVariable/CSharpIntroduceVariableService_IntroduceLocal.cs

```diff
@@ -53,7 +53,7 @@ internal partial class CSharpIntroduceVariableService
             var text = await document.Document.GetTextAsync(cancellationToken).ConfigureAwait(false);
             if (!text.AreOnSameLine(containerToGenerateInto.GetFirstToken(), containerToGenerateInto.GetLastToken()))
             {
-                declarationStatement = declarationStatement.WithAppendedTrailingTrivia(SyntaxFactory.ElasticCarriageReturnLineFeed);
+                declarationStatement = declarationStatement.WithTrailingTrivia(SyntaxFactory.CarriageReturnLineFeed);
```

Comment:
I believe "Elastic" means that formatter should keep it consistent with other file "EOL"s.

Reply:
The expectation is as @Youssef1313 says: the formatter should do the right thing if you use `ElasticCarriageReturnLineFeed`

Reply:
Thanks for the update, I'll take another look shortly.

Reply:
❗ Yes, this needs to use elastic.

Reply:
📝 As an alternative to using elastic trivia, it would be sufficient to expand the test to cover both the CRLF and LF line ending scenarios, demonstrating that the code fix does not require an elastic token to support both cases.